src/test/java/com/mealmentor/enterprise/EnterpriseApplicationTests.java

```diff
@@ -40,7 +38,7 @@ void contextLoads() {
     }
 
     @Test
-    void saveMealItem_validateReturnMealItemWithLasagnaMondayDinner() throws Exception  {
+    void saveMealItem_validateReturnMealItemWithLasagnaMondayDinner() throws Exception {
```

Comment:
Removed whitespaces as part of the good clean coding conventions.

To learn more: https://www.baeldung.com/java-clean-code